pennylane_ionq/device.py

```diff
@@ -298,3 +327,98 @@ def generate_samples(self):
         samples = np.repeat(np.arange(number_of_states), counts)
         np.random.shuffle(samples)
         return QubitDevice.states_to_binary(samples, self.num_wires)
+
+
+# Specific Backends

```

Comment:
Let's stick to `qml.device(“ionq.qpu”, backend=“backend-name”, api_key=api_key, wires=2)` and avoid adding specific subclasses for each hardware unit

Reply:
alright that sounds good! itll definitely be easier to maintain this way :)